Add Licensing section (#5624)
DevExpress/devextreme-documentation: concepts/Common/Licensing/15 License Key Expiration and How to Update.md

> A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 

Reviewer: **minor**
As for me, the main idea is that these things are coupled, not that they are provided
```suggestion
A license key is bound to a major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 
```

> A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 

Reviewer: Do not mention beta - we've just decided that no key will be available for betas
```suggestion
A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, etc. When switching to a new major version, the license key must be updated. 
```

> A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 

Reviewer: ```suggestion
A license key is supplied with each specific major version. It's valid for all minor and patch updates within this major version regardless of when they are published. When upgrading to a new major version, the license key must also be updated. 
```

> A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 

Reviewer 2: We rethought license key working with  pre-release version. Let's reorganize this sectionю All Ilya's remarks took into account.
Examples for different versions added because DevExtreme versioning is not the same with semver.

> A license key is bound to each specific DevExtreme major version, for example, 23.1 and 23.2 are two different major versions. The license key will work with any minor versions released within one major, for example 23.1.3 and 32.1.5 are two minors within 23.1 major version. 
>
> The license key itself does not expire, which complies with the terms of the perpetual license. If you remain on the same major version and do not update DevExtreme, even if you perform a renew on your DevExtreme license, then you do not need to update the license key. License keys for all Devextreme versions available to you will always be in the Download Manager even after expiration of the Devextreme license.
>
> If you are updating DevExtreme to a new major version, the license key must be updated to one that will be bound to this new major.
> We do not provide license keys for pre-release versions (such as EAP, beta, CTP etc), as these versions are not suitable for use for commercial purposes or production using.
> 
> [note] License keys for newer versions of DevExtreme can be used for all versions of DevExtreme released earlier.

> [note] License keys for newer versions of DevExtreme can be used for all versions of DevExtreme released earlier.

Reviewer: ```suggestion
[note] License keys for newer versions of DevExtreme can be used for all major versions of DevExtreme released earlier.
```
